src/app/rate/page.js

```diff
@@ -1,6 +1,25 @@
+"use client";
+
+import { useEffect } from "react";
 import Form from "./Form";
+import { useRouter } from "next/navigation";
+import { account } from "@/config/appwrite-client";
 
 export default function Rate() {
+  const router = useRouter();
+
+  const getUser = async () => {
+    try {
+      const user = await account.getSession("current");
+    } catch (e) {
+      router.push('/auth/login');
```

Comment:
Update quotes to be consistent with project

```suggestion
      router.push("/auth/login");
```